daemon/algod/api/client/restClient.go

```diff
@@ -422,7 +422,7 @@ func (client RestClient) AccountInformation(address string) (response v1.Account
 func (client RestClient) AccountInformationV2(address string, includeCreatables bool) (response generatedV2.Account, err error) {
 	var infoParams accountInformationParams
 	if includeCreatables {
-		infoParams = accountInformationParams{Exclude: "", Format: "json"}
+		infoParams = accountInformationParams{Exclude: "none", Format: "json"}
```

Comment:
Isn't this backwards?
if includeCreatables == true then "all" else"none"?

Reply:
you might want to argue that the parameter to `AccountInformationV2` should be `excludeCreatables` instead of `includeCreatables`.. but I think that the logic here is correct otherwise.